include/openssl/asn1t.h.in

```diff
@@ -657,6 +663,8 @@ typedef struct ASN1_EXTERN_FUNCS_st {
     ASN1_ex_d2i *asn1_ex_d2i;
     ASN1_ex_i2d *asn1_ex_i2d;
     ASN1_ex_print_func *asn1_ex_print;
+    ASN1_ex_new_ex_func *asn1_ex_new_ex;
+    ASN1_ex_d2i_ex *asn1_ex_d2i_ex;
```

Comment:
Can you add new members to a public structure? Hmm, I suppose it is OK if they are added at the end and we aim just for an API compatibility.

Reply:
Yes - I think this is fine. We do not guarantee ABI compatibility.